src/mon/ConfigMonitor.cc

```diff
@@ -542,6 +542,15 @@ bool ConfigMonitor::prepare_command(MonOpRequestRef op)
     name = ConfFile::normalize_key_name(name);
     
     if (prefix == "config set" && !force) {
+
+      std::string specialSymbols = "!@#$%^&*";
```

Comment:
no need for the (non const!) variable. Just use a string literal (or just C string) in the call to find_first_of()

Reply:
Hey @ronen-fr I have made the changes you suggested